File: AzW.Web.Portal/src/models/ASE.js
import ResourceType from './ResourceType'
export default class ASE
{
    constructor (){
        this.GraphModel = {
            Id: '',
            ResourceType: ResourceType.ASE(),
            DisplayName: ''
        };

        this.ProvisionContext = {
            ResourceType: ResourceType.ASE(),
            Name: '',
            Location: '',
            ResourceGroupName: '',
            VNetName: '',
            SubnetName: '',
            IsInternalASE: true
        };
        this.CalculatorContext = {
            IsFree: false,
            Tier: ''
        }
    }
}

File: Portal.Services/AzW.Model/Provision/ASE.cs
namespace AzW.Model
 {
     public class ASE : Resource
     {
          public string Name { get; set; }
          public string Location { get; set; }
          public string ResourceGroupName { get; set; }
          public string VNetName { get; set; }
          public string SubnetName { get; set; }
          public bool IsInternalASE { get; set; }
     }
 }